CONTRIBUTING.rst

```diff
@@ -16,22 +16,40 @@ We love pull requests from everyone! Here's a quick guide to improve the code:
 
     pip install -r requirements-dev.txt
 
-4. Make sure all tests pass:
+4. Pull the image for ROS integration testing:
+
+::
+
+    docker pull gramaziokohler/integration-tests-bridge
+
```

Comment:
This step could be removed, since it happens automatically on the next step if the image is not downloaded.

Reply:
right, of course